packages/dnd-timeline/src/hooks/useItem.tsx

```diff
@@ -193,8 +193,10 @@ export default function useItem(props: UseItemProps) {
 
 			setDragDirection(null);
 
-			draggableProps.node.current.style.width = `${width}px`;
-			draggableProps.node.current.style[sideStart] = `${deltaXStart}px`;
+			if (draggableProps.node.current?.style) {
```

Comment:
Why is this not caught by typescript? Why is `draggableProps.node.current` not marked as optionally null? 
I suspect there is a deeper type problem. Maybe a check is not performed before entering this callback, or the draggable component is un-rendered while this function is called...

Can you please include the implementation in which you encountered this error, so that we can further investigate?

Reply:
In line 166, this is checked to be defined. Are you saying it changed while running the code?

Reply:
Maybe try to call onResizeCallback after?